src/screens/signup/signup2.tsx

```diff
@@ -1,32 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { SafeAreaView, TouchableOpacity } from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 
 import { NavigationInterface } from '../types';
 import { useThemeContext } from '../../theme';
 
-import { Container, Welcome, Prompt, NavigationView, Pagination, OfText } from './styles';
+import Button from '../../components/button';
+import Input from '../../components/input';
+
+import {
+  Container,
+  Welcome,
+  Prompt,
+  NavigationView,
+  Pagination,
+  OfText
+} from './styles';
 
 interface SignupScreenTwoProp extends NavigationInterface {
   testID?: string;
 }
 
 export default function SignUpScreenTwo(props: SignupScreenTwoProp) {
   const { colors, fonts } = useThemeContext();
+
+  const [email, setEmail] = useState('clairetamara08@gmail.com');
+  const [values, setValues] =  useState({
+      firstName: '',
+      lastName: ''
+  })
+
   return (
     <Container>
-     <SafeAreaView>
-         <NavigationView>
-            <TouchableOpacity onPress={() => props.navigation.goBack()}>
+      <SafeAreaView>
+        <NavigationView>
```

Comment:
For header navigations checkout in-app custom header for navigation and other stuffs,

See examples from regiment screen settings screen or hair care screen